### PR TITLE
ci: expand mypy coverage

### PR DESCRIPTION
**Pull request by tseaver**

### CODE REVIEW

File: google/cloud/bigtable_admin_v2/services/bigtable_instance_admin/async_client.py [hunk 1]
```diff
@@ -199,9 +199,7 @@ async def create_instance(
         parent: str = None,
         instance_id: str = None,
         instance: gba_instance.Instance = None,
-        clusters: Sequence[
-            bigtable_instance_admin.CreateInstanceRequest.ClustersEntry
-        ] = None,
+        clusters: Dict[str, gba_instance.Cluster] = None,
```
Comment (tswast) [inline google/cloud/bigtable_admin_v2/services/bigtable_instance_admin/async_client.py:202]: This is inconsistent with the docstring. Are we removing types from docstrings now like we've been in the BigQuery client?

File: google/cloud/bigtable_admin_v2/services/bigtable_instance_admin/client.py [hunk 2]
```diff
@@ -430,9 +430,7 @@ def create_instance(
         parent: str = None,
         instance_id: str = None,
         instance: gba_instance.Instance = None,
-        clusters: Sequence[
-            bigtable_instance_admin.CreateInstanceRequest.ClustersEntry
-        ] = None,
+        clusters: Dict[str, gba_instance.Cluster] = None,
```
Comment (tswast) [inline google/cloud/bigtable_admin_v2/services/bigtable_instance_admin/client.py:433]: Inconsistent with docstring

File: google/cloud/bigtable_admin_v2/services/bigtable_instance_admin/async_client.py (hunk 1 above)
Comment (tswast) [reply google/cloud/bigtable_admin_v2/services/bigtable_instance_admin/async_client.py:202]: Just realized this is generated code. Perhaps the `owlbot` rule could be tweaked to do the same replacement in the docstring?

Is there a bug open for the code generator for this replacement?

Comment (tseaver) [reply google/cloud/bigtable_admin_v2/services/bigtable_instance_admin/async_client.py:202]: https://github.com/googleapis/gapic-generator-python/issues/689

Comment (tseaver) [reply google/cloud/bigtable_admin_v2/services/bigtable_instance_admin/async_client.py:202]: @tswast f7441f4 fixes up the docstring to match.

**Review by tswast — APPROVED** [review]

**Pull request merged**